core/templates/dev/head/profile/Profile.js

```diff
@@ -31,22 +31,83 @@ oppia.controller('Profile', [
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
+      $scope.username = {
```

Comment:
I will be keeping the original code as is, as the stat format is needed for there for be a popover containing the entire username.